mscore/scoreview.cpp

```diff
@@ -548,7 +548,9 @@ void ScoreView::moveCursor(const Fraction& tick)
             qreal x2;
             Fraction t2;
             Segment* ns = s->next(SegmentType::ChordRest);
-            if (ns && ns->visible()) {
+            while (ns && !ns->visible())
```

Comment:
I suppose I should have created a nextVisible() function, but it didn't seem at first like we'd be actually using it much.

Reply:
Or a version of nextActive() that takes a SegmentType as a parameter.